src/Admin/IsAdherentManager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use AppBundle\Entity\Adherent;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class IsAdherentManager
+{
+    private $registry;
+
+    public function __construct(RegistryInterface $registry)
+    {
+        $this->registry = $registry;
+    }
+
+    public function isAdherent(string $email): bool
```

Comment:
You can add this method in the `AdherentManager` instead of create a new `IsAdherentManager`